database/seed.js

```diff
@@ -0,0 +1,57 @@
+const faker = require('faker');
+const Review = require('../database/index.js');
+
+const hereForArr = ['', 'Breakfast', 'Brunch', 'Lunch', 'Dinner', 'Dessert', 'Coffee or tea', 'Snacks', 'Drinks', 'Late night food', 'Other'];
+const travelerTypeArr = ['Couples', 'Family', 'Friends', 'Business', 'Solo'];
```

Comment:
I see you put an empty string in your hereforArr but not one in your travelerTypeArr.  Is the empty string indicating an option that is not filled in by the reviewer?

Reply:
Great question! I put in '' to account for the fact that it's an optional field. However, travelerType is also optional -- I need to add '' there too. Thanks for catching!